.github/workflows/scheduled.yml

```diff
@@ -3,7 +3,7 @@ name: Scheduled Build
 on:
   push:
     branches:
-      - main
+      - Integrate_Unity_CMock
 
```

Comment:
Don't change the branch name because this is for main branch

1- If you did that because you wanted to test CI for your branch its okay, but then you should have reverted it back
2-please rebase to main because I've created workflow for pull request, so an automated checks and tests will be running on your pull request you don't have to touch scheduled